src/encode.cpp

```diff
@@ -1035,7 +1136,7 @@ void encodeOp(State &st, mlir::tosa::Conv2DOp op, bool) {
 
   auto output = Tensor::mkInitializedLambda(
                   elemTy, move(outDims), move(outInd), 
-                  biasf.add(tf)
+                  tf.add(biasf)
```

Comment:
Why is this changed?

Reply:
to be same as the tosa spec

Reply:
Okay, fine.
FYI: since fp add is commutative, they are equivalent.